src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -348,11 +350,20 @@ class GTGRunner(val masterURL: String,
     val subgraphs = sc.sparkContext.parallelize(edgeListRDDIndexed.toSeq)
       .map(MCCOps.mapEdgesToBuckets(_, maxParitionSize, buckets))
       .groupByKey()
-    val subgraphsFound = MCCOps.findSubgraphsIteratively(subgraphs, 1, maxParitionSize, minGraphLength, sc.sparkContext)
+    val subgraphsFound = MCCOps.findSubgraphsIteratively(subgraphs, 1, maxParitionSize,
```

Comment:
Yes.
In general lets avoid this pattern.
"collect -> some transform on collect -> parallelize"
Instead, lets keep the data on the worker side as much as possible.

Collect's will usually result in a large data structure, and parallelizing will generate the warning.